ui/js/page/filePage/view.jsx

```diff
@@ -116,7 +116,7 @@ class FilePage extends React.PureComponent {
                   : uriIndicator}
               </div>
               <div className="card__actions">
-                <FileActions uri={uri} />
+                <FileActions uri={uri} claim={claim} />
```

Comment:
It is(by resolving it). But I think re-resolving the same URL, would be a useless overhead, when we could simply use an existing data-set. This also helps the app to remain fast and fluid in its operation. I suggest keeping this as is. What's your stance on this?